lib/config/sfApplicationConfiguration.class.php

```diff
@@ -128,7 +128,15 @@ public function initConfiguration()
         }
 
         // error settings
-        ini_set('display_errors', $this->isDebug() ? 'on' : 'off');
```

Comment:
Does it mean that the `$this->isDebug()` is not used anymore to drive the configuration for error displaying?

Reply:
Good point, the old logic did turn off `display_errors` if `isDebug()` is `false`. I've changed the `if`-case to deactivate it as well. The activation in case log_errors or error_log is not configured will therefore only be done, if in debug mode. Current sf does it only if `Debug::enable()` is called.

Reply:
Nice, thanks! Would you please add a line of comments to the changed code? just to be sure that reading it will remind us why we have that `if-else` when dealing with logs :)

LGTM otherwise :+1:

Reply:
done, is it ok?